test-data/index.js

```diff
@@ -0,0 +1,2 @@
+const limits = require('mapbox-upload-limits');
+console.log(JSON.stringify(limits));
```

Comment:
This can probably be removed?

Reply:
that bit is tied to this bit,
https://github.com/mapbox/mapbox-upload-limits/blob/bugfix_12/test.js#L38-L48